src/socket/tcp.rs

```diff
@@ -530,6 +546,39 @@ impl<'a> Socket<'a> {
         }
     }
 
+    /// Set an algorithm for congestion control.
+    ///
+    /// The default setting is `CongestionControl::None`, indicating that no congestion control is applied.
+    /// Options `CongestionControl::Cubic` and `CongestionControl::Reno` are also available.
+    ///
+    /// `CongestionControl::Cubic` represents a modern congestion control algorithm designed to
+    /// be more efficient and fair compared to `CongestionControl::Reno`.
+    /// It is the default choice for Linux, Windows, and macOS.
+    ///
+    /// `CongestionControl::Reno` is a classic congestion control algorithm valued for its simplicity.
+    /// Despite having a lower algorithmic complexity than `Cubic`,
+    /// it is less efficient in terms of bandwidth usage.
+    pub fn set_congestion_control(&mut self, congestion_control: CongestionControl) {
+        use congestion_controller::*;
+
+        self.congestion_controller = match congestion_control {
+            CongestionControl::None => GenericController::None,
+            CongestionControl::Cubic => GenericController::Cubic(cubic::Cubic::default()),
+            CongestionControl::Reno => GenericController::Reno(reno::Reno::default()),
+        }
+    }
+
+    /// Return the current congestion control algorithm.
+    pub fn get_congestion_control(&self) -> CongestionControl {
```

Comment:
This should be called `fn congestion_control` when following our naming scheme for accessors.

Reply:
fixed